nengo_viz/swi.py

```diff
@@ -465,7 +465,7 @@ def handle_error(self, request, client_address):
                 exc_value.args[0] == errno.EBADF):
             return  # Probably caused by a server shutdown
         else:
-            print exc_type, exc_value, dir(exc_value)
+            print(exc_type, exc_value, dir(exc_value))
```

Comment:
Maybe we should do it elsewhere too. Or actually I think we should get rid of all print statements and replace them with something which gives us more control about when and where to stuff is printed (but not before the summer school).

The print statement here will behave differently depending on the existence of the `__future__` import (and without depending on the Python version, I suppose):
```python
In [1]: print("a", "b")
('a', 'b')

In [2]: from __future__ import print_function

In [3]: print("a", "b")
a b
```